docs/index.html

```diff
@@ -759,8 +759,8 @@ <h1 id="api-humandiff">Difference for Humans</h1>
 
 echo Carbon::now()->addSeconds(5)->diffForHumans();            // 5 seconds from now
 
-echo Carbon::now()->subDays(24)->diffForHumans();              // 3 weeks ago
-echo Carbon::now()->subDays(24)->diffForHumans(null, true);    // 3 weeks
+echo Carbon::now()->subDays(24)->diffForHumans();              // 3 weeks 3 days ago
```

Comment:
This is a bug.  Will fix it in a PR.

Reply:
I fixed it in #1103  It didn't break any tests because it wasn't being tested.  I also added some to test the $parts param.